jdbc/pom.xml

```diff
@@ -35,6 +35,11 @@ LICENSE file.
       <artifactId>openjpa-jdbc</artifactId>
       <version>${openjpa.jdbc.version}</version>
     </dependency>
+    <dependency>
+      <groupId>postgresql</groupId>
+      <artifactId>postgresql</artifactId>
+      <version>9.1-901-1.jdbc4</version>
```

Comment:
Is there a reason to use this version? (see latest list here: https://jdbc.postgresql.org/download.html)

Reply:
Yeah, I should have realized this is old.
I googled the list and the first result gave out this version.

WIll be more careful about these versions.
Thanks.